src/Integration/LocalServices/ProjectSystemHelper.cs

```diff
@@ -237,7 +244,7 @@ private static IEnumerable<IVsHierarchy> EnumerateProjects(IVsSolution solution)
             }
         }
 
-        public static IEnumerable<Guid> GetAggregateProjectKinds(IVsHierarchy hierarchy)
+        private static IEnumerable<Guid> GetAggregateProjectKinds(IVsHierarchy hierarchy)
```

Comment:
Only caller is `ProjectSystemHelper.IsKnownTestProject` (and we're not using any reflection), so this access modifier change should be safe.